### DISCUSSION TITLE
Does package support base 64 images and arabic language or Rtl language?

### DISCUSSION THREAD

**Discussion by ahmedzakaria432** [richard-scryber/scryber.core#99] [Q&A]
I am trying to use package to generate pdf that contains Arabic language , all Arabic words displayed as ?? (question marks) 
is there any way to resolve this issue,
also base 64 images not displayed

**Comment by richard-scryber** [comment]
Hi Ahmed, I have previously tried this as I know many people would benefit from it.
Firstly the font used must support your beautiful characters (and the default Helvetica or Times do not).

However I know your language has many ligatures (joining the letters together). Without that, it doesn't make sense. Currently we don't support that. It may be something we can look into in the not too distant future, and would be great to have someone to check and validate. My Arabic is just not there :-)

**Comment by richard-scryber** [comment]
Inline data urls with Base64 images should be supported
